CodeChat_Server/easyinstallnotes.rst

```diff
@@ -0,0 +1,118 @@
+**************************
+Easy Installation and use
+**************************
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+
+To install the CodeChat System, install the CodeChat Server and a CodeChat extension or plugin, along with optional external renderers you desire.
+
+.. _install CodeChat Server:
+
+CodeChat Server Installation Notes
+==================================
+
+.. note:: 
+ 
+ the idea behind our entire project is to get this instruction list as short at possible
+
+Windows
+-------
+#.  Go to this `link <https://github.com/JoeKenn1118/CodeChat_system/releases/tag/v0.0.1>`_ and download the Powershell script (``windows_installer.ps1``)
+
+#. Open up Powershell in Admin mode
+
+#. ``cd`` to the file location
+
+#. Run ``windows_installer.ps1``
+
+
+
+Linux
+-----
+#.  `Open a terminal <https://www.howtogeek.com/howto/22283/four-ways-to-get-instant-access-to-a-terminal-in-linux/>`__.
+
+#.  Highlight this entire command: `wget <https://raw.githubusercontent.com/JoeKenn1118/CodeChat_system/master/CodeChat_Server/codechat.sh && bash codechat.sh>`_ and press Ctrl-C to copy. Go to the terminal window and press Ctrl-Shift-V. Press Enter.
```

Comment:
A bit ugly -- should be `wget blah | bash` or something similar. Avoid actually saving the file.